app/controllers/TaskController.scala

```diff
@@ -107,8 +106,10 @@ class TaskController @Inject() (val messagesApi: MessagesApi) extends Controller
   private def parseResultToSkeletonTracingFox(parseResult: NmlService.NmlParseResult): Fox[SkeletonTracing] = parseResult match {
     case NmlService.NmlParseFailure(fileName, error) =>
       Fox.failure(Messages("nml.file.invalid", fileName, error))
-    case NmlService.NmlParseSuccess(fileName, tracing) =>
-      Fox.successful(tracing.asInstanceOf[SkeletonTracing])
+    case NmlService.NmlParseSuccess(fileName, (Left(skeletonTracing), description)) =>
```

Comment:
This change fixes a crash that happened before, due to changes in the ParseResult type. It is not directly related to the title of this PR